src/index-debug.html

```diff
@@ -24,38 +24,17 @@
 <html>
 <head>
     <meta charset="utf-8">
-    <title>Photoshop Design Space (optimized build)</title>
-    <link rel="stylesheet" type="text/css" href="style/main.css" />
-    <script src="js/require.js"></script>
-    <script>
-        var locale = window.navigator.language,
-            parts = locale.match(/\w+/),
-            language = parts.length > 0 && parts[0];
+    <title>Photoshop Design Space (development build)</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <script type ="text/javascript" src="./externalDeps.js"></script>
```

Comment:
Also, I don't understand the motivation behind splitting out the external dependencies into a separate bundle.